extras/agc-minimal-permissions/lib/policy-statements.ts

```diff
@@ -825,6 +842,7 @@ export class AgcPermissions {
                 effect: Effect.ALLOW,
                 actions: actions(svc,
                     "CreateHostedZone",
+                    "ListHostedZonesByName",
```

Comment:
Do we need the route53 policy now that we have removed the cloudmap in #122. I don't think route53 was used elsewhere

Reply:
Possibly not. I'll want to test this after a new release that incorporates #122.